### DISCUSSION TITLE
The difference between Unikraft and MiniOS.

### DISCUSSION THREAD

**Discussion by hack3rcon** [unikraft/unikraft#57] [Q&A] [answered]
Hello,
I don't know here is a good place for asking questions or not. I have a general questions:
What is the difference between the Unikraft and MiniOS?

Thanks.

**Comment by nderjung** [comment, marked as answer]
From @skuenzer previously posted on [our mailing-list](https://lists.xenproject.org/archives/html/minios-devel/2020-09/msg00040.html):

 > Both projects are Unikernels, where Mini-OS is the older project. Its original purpose was to demonstrate the Xen hypervisor interface. As far as I know, it is used today in some Xen deployments as basis for stub domains (e.g., QEMU) and running the XenStore separately from Dom0. Main obvious difference to Unikraft is that MiniOS works just with Xen.
 > 
 > With Unikraft we try to provide a framework for building specialized Unikernels across different platforms, where Xen or KVM are just two of them. The project is younger and we are working heavily to develop a library pool where you basically pick and choose OS functionality that you need for your Unikernel.

**Comment by razvand** [comment]
@hack3rcon, since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
